09-ncvs-vignette.Rmd

```diff
@@ -677,7 +679,7 @@ pers_est_df %>%
     locations = cells_column_spanners(spanners = "viol_span")
   ) %>%
   tab_footnote(footnote = "Excludes persons of Hispanic origin",
-               locations = cells_stub(rows = Level != "Hispanic")) %>%
+               locations = cells_stub(rows = Level != "Hispanic")) %>% 
```

Comment:
This is putting a reference on ALL other labels in the table and not just all other labels in the Race section.  May need to call out the specific race/ethnicity items here.